### DISCUSSION TITLE
Error in the dashboard clicking the fullfillment button

### DISCUSSION THREAD

**Discussion by DidoMarchet** [mirumee/saleor-dashboard#2853] [General] [answered]
Hi,
I have encountered an error when I click the FULFILL button in the Saleor dashboard:

dashboard v3.1.0-dev
core v3.5.0-a

I would like to know if there's a way to debug it easly or if this error has already been recognised.
If an upgrade of the system is required are the old SDK https://github.com/saleor/saleor-sdk still supported?
Because this project works with them.

Thanks in advance and kind regards,

Davide

<img width="1302" alt="Screenshot 2022-12-07 at 09 31 34" src="https://user-images.githubusercontent.com/38940340/206128115-60a21107-3687-4d9c-840b-ba4c65cffe35.png">

<img width="640" alt="Screenshot 2022-12-07 at 09 21 11" src="https://user-images.githubusercontent.com/38940340/206128144-849e4989-540e-41e3-9692-77c3ab754531.png">

**Comment by DidoMarchet** [comment]
Hi!
any ideas for this generic error?

If we try to refresh the page few times it works.

Thanks in advance and kind regards,

Davide

**Comment by DidoMarchet** [comment]
Hi,
do you have any suggestions?
They report an other error (in refund for example)
and we don't know the cause.

![Screen Shot 2022-12-13 at 2 37 47 PM](https://user-images.githubusercontent.com/38940340/207540798-9008ea7d-88ab-4f54-9418-cc5c5b60ab50.png)

Thanks in advance and kind regards,

Davide

**Comment by karolkielecki** [comment, marked as answer]
Hi @DidoMarchet. 3.1 dash + 3.5 core -- I'm not sure if these versions are compatible. We recommend updating Saleor to the latest versions. If you still receive such errors, please let us know.

**Reply by DidoMarchet** [reply]
Hi @karolkielecki  we provide to update.
The only thing is this ecommerce works with https://github.com/saleor/saleor-sdk .
Are them still compatible right? 

Thanks in advance and kind regards,

Davide

**Reply by karolkielecki** [reply]
Yes, SDK is still compatible with the newer versions. The dashboard still uses it.
